package.json

```diff
@@ -15,6 +15,7 @@
     "url": "https://github.com/Sage-Bionetworks/Synapse-React-Client"
   },
   "dependencies": {
+    "@apidevtools/json-schema-ref-parser": "^9.0.9",
```

Comment:
Same package was moved to a different namespace. No breaking changes that affect us as far as I could tell.